src/components/App/App.js

```diff
@@ -1,29 +1,42 @@
 import React, { useState, useEffect } from 'react'
-import { fakeData } from '../../data'
+// import { fakeData } from '../../data'
 import { Switch, Route } from 'react-router-dom'
 import Navbar from '../Navbar/Navbar'
 import Home from '../Home/Home'
 import './App.scss'
 import Dashboard from '../Dashboard/Dashboard'
-import Context from '../../Context'
+import UsersContext from '../../Context/UsersContext'
+import ErrorContext from '../../Context/ErrorContext'
+import { getUsers } from '../../utilities/apiCalls'
 
 export default function App() {
 
   const [users, setUsers] = useState([])
+  const [error, setError] = useState('')
 
-  useEffect(() => {
-      setUsers(fakeData.data)
+  useEffect(() => {  
+    const fetchUsers = async () => {
+      try {
+        const response = await getUsers()
+        setUsers(response.data)
+      } catch (err) {
+        setError(err)
+      }
+    }
+    fetchUsers()
   }, [])
 
   return (
-    <Context.Provider value={users}>
-      <main>
-          <Navbar />
-        <Switch>
-          <Route exact path='/' component={Home} />
-          <Route exact path='/dashboard' component={Dashboard} />
-        </Switch>
-      </main>
-    </Context.Provider>
+    <ErrorContext.Provider value={error}>
+      <UsersContext.Provider value={users}>
+        <main>
+            <Navbar />
+          <Switch>
+            <Route exact path='/' component={Home} />
+            <Route exact path='/dashboard' component={Dashboard} />
+          </Switch>
+        </main>
+      </UsersContext.Provider>
+    </ErrorContext.Provider>
```

Comment:
It's been fun learning context with you. I wonder how readability is maintained when there are many contexts. Something to think about. This looks good so far.